.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -23,8 +23,20 @@ A clear and concise description of what you expected to happen.
 If applicable, add config files to help explain your problem.
 
 **Environment**
-- Cloud provider: [AWS | Azure | Any]
+- Cloud provider: [AWS | Azure | All | None]
 - OS: [e.g. Ubuntu 18.04.4 LTS, you can use `cat /etc/os-release`]
 
+**epicli version**: [`epicli --version`]
+
 **Additional context**
 Add any other context about the problem here.
+
+---
+
+**DoD checklist**
+
+* [ ] Changelog updated (if affected version was released)
+* [ ] Automated tests passed (no regression)
+* [ ] Case covered by automated test (if possible)
+* [ ] Idempotency tested
+* [ ] Documentation updated / doesn't need to be updated
```

Comment:
I remember that there was an issue caused by Elasticsearch version. WDYT about adding `COMPONENTS.md updated / doesn't need to be updated` here?